server/routes/createPlan/checkYourAnswers/checkYourAnswersController.test.ts

```diff
@@ -17,24 +17,29 @@ describe('CheckYourAnswersController', () => {
     lastName: 'mock_lastName',
   }
 
-  req.params.id = 'A1234BC'
+  const prisonNumber = 'A1234BC'
+  req.params.id = prisonNumber
   req.params.mode = 'new'
   const { id } = req.params
 
   const mockData = {
-    id: 'A1234BC',
+    id: prisonNumber,
     prisoner: plainToClass(PrisonerViewModel, req.context.prisoner),
     record: { hopingToGetWork: 'YES' },
```

Comment:
Unfortunately only one of the fields in the `record` is being populated, which means none of the mapping is being currently being tested. I changed it to populate most of the fields, but that breaks the `get()` method tests and they're difficult to fix without using mocked mappers.

We can test this indirectly via integration tests, but as I said in my other comment, I'd like to consider re-writing this class once we're calling our API.